src/Microsoft.Identity.Client/Platforms/Android/AuthenticationContinuationHelper.cs

```diff
@@ -51,7 +51,9 @@ public static class AuthenticationContinuationHelper
         [CLSCompliant(false)]
         public static void SetAuthenticationContinuationEventArgs(int requestCode, Result resultCode, Intent data)
         {
-            RequestContext requestContext = new RequestContext(null, new MsalLogger(Guid.Empty, null));
+            // TODO(migration): how can a public static method get access to the proper ClientRequestBase to wire into the logger and appropriate requestcontext?
+            // Can we move this call to be somewhere on the ClientApplicationBase or something else that's wired into that?
```

Comment:
>// Can we move this call to be somewhere on the ClientApplicationBase or something else that's wired into that? [](start = 12, length = 111)

This is an Android only thing used to handle the call backs from the webview window... today it's required to listen to this. This is not something for the ClientAppBase imo. App developers can calls this themselves